.travis.yml

```diff
@@ -1,7 +1,25 @@
+sudo: required
+
 language: node_js
 node_js:
-  - "stable"
+- "node" 
+
+services:
+- docker
+
+after_success:
+- if [ "$TRAVIS_BRANCH" == "master" ]; then
+  docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD";
+  docker push kevinchu/first-blog-mern;
+  fi
 
 script:
-  - npm run lint
-  - npm run test
+- npm run lint
+- npm run test
+- docker build -t kevinchu/first-blog-mern .
+
+env:
```

Comment:
Personally I would put `env` near the top of the file (maybe after `services`), but not a real issue